css/_uswds-theme-custom-styles.scss

```diff
@@ -420,6 +420,7 @@ img {
     bottom: 0;
     position: sticky;
     z-index: z-index(500);
+    pointer-events: none;
```

Comment:
Is there any value in moving these `pointer-event` rules out of the media queries and into the base style definitions? Would we ever want them to _not_ be present?

Reply:
I picked the media query because in general, I think it's unwise to change the native click behaviors anywhere you don't _have_ to, and this limited the impact to just the screen width range where the sticky positioning causes the click-blocking overlap, and it doesn't affect native behaviors with normal positioning at all the other screen widths.

Reply:
Thanks for clearing that up! This is a good point and I'm happy to leave it where it is